docs/modules/ROOT/pages/index.adoc

```diff
@@ -990,51 +990,53 @@ For that reason, might make sense to move proof up one level from its current pl
 
 ===== Binding to C2PA asset
 
-The `credentialSubject` field MUST contain a `boundTo` entry, which MUST be the JSON serialization of the `signer_payload` data structure presented for signature. All bytestring entries in the `signer_payload` data structure (i.e. hash entries in hashlink URL data structures) MUST be converted to the corresponding Base-64 plain-text strings.
+The `credentialSubject` field MUST contain a `c2paAsset` entry, which MUST be the JSON serialization of the `signer_payload` data structure presented for signature with the following adaptations:
```

Comment:
should we just call it signer_payload as well in the VC since that's what it is ?